src/components/products/tokenMetadata/index.js

```diff
@@ -64,7 +64,7 @@ export const tokenMetadata = {
               href: '/token-metadata/delegates',
             }, // Delegate + Revoke + Delegated transfer and burn.
             { title: 'Locking Assets', href: '/token-metadata/lock' },
-            { title: 'Programmable NFTs', href: '/token-metadata/pnfts' },
+            { title: 'Programmable NFTs (pNFT)', href: '/token-metadata/pnfts' },
```

Comment:
Nit: I think technically you should do pNFTs since the word you were turning into an acronym was plural?